CSS/components/button.css

```diff
@@ -0,0 +1,17 @@
+.button--answer {
```

Comment:
Using a class name 'button' on a `<button>` element is not very descriptive. Maybe you can think of something more descriptive like:
'card__button' - the general button style rules
'card__button--show' - class/rule when button is shown
'card__button--hide' - class/rule when button is hidden